cves/CVE-2016-4430.yml

```diff
@@ -119,20 +126,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
-  contest:
+  answer: |
+    apache confluence, CVE page references
```

Comment:
I'm not sure what this is, can you elaborate?

Reply:
Thank you! I have replaced this with a more detailed explanation beyond the reference to sites.